osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/RsaEncryptionService.java

```diff
@@ -267,7 +268,7 @@ private RSAPrivateKey getRsaPrivateKey(final PrivateKey privateKey) {
      * @return a public key for algorithm {@value #ALGORITHM}, or {@code null}
      *         if the given {@code filename} is blank.
      */
-    public static PublicKey readPublicKeyFromFile(final String filename) {
+    public static PublicKey readPublicKeyFromFile(final String filename) throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)